packages/component-carousel/package.json

```diff
@@ -31,13 +31,16 @@
     "postbuild": "cp ./types/main.d.ts ./dist/main.d.ts",
     "test": "jest --config=./jest.config.js --passWithNoTests --silent --coverage",
     "start:dev": "webpack-dashboard -- webpack serve -c webpack/webpack.dev.js",
-    "storybook": "start-storybook -s ./dist -p 9009",
+    "storybook": "start-storybook -s ./dist -p 9040",
     "build-storybook": "build-storybook -s ./dist",
     "jsdoc": "jsdoc -c jsdoc.config.js",
     "predocs": "mkdir -p ./docs",
     "docs": "jsdoc2md --no-cache -c jsdoc.config.js --files ./src/components > ./docs/README.props.md",
     "postdocs": "node ../../scripts/process-readme-props.js"
   },
+  "peerDependencies": {
+    "postcss": "^8.4.19"
+  },
```

Comment:
@scott-williams-az I noticed `postcss` is defined as both a peerDependency and a devDependency. Is the peerDependency requirement necessary?

Reply:
I'll double check